src/clue.js

```diff
@@ -1,26 +1,216 @@
 // ITERATION 1
 
 // Suspects Array
+let mrGreen = {
+  firstName: "Jacob ",
+  lastName: "Green",
+  occupation: "Entrepreneur",
+  age: 45,
+  description: "He has a lot of connections",
+  image:
+    "https://carboncostume.com/wordpress/wp-content/uploads/2017/10/mrgreen3.jpg",
+  color: "green ",
+}
+let drOrchid = {
+  firstName: "Doctor",
+  lastName: "Orchid",
+  occupation: "Scientist",
+  age: 26,
+  description: "PhD in plant toxicology. Adopted daughter of Mr. Boddy",
+  image:
+    "https://images2.minutemediacdn.com/image/upload/c_crop,h_1124,w_2000,x_0,y_50/v1560965041/shape/mentalfloss/586055-hasbro.jpg?itok=x4QQWTxa",
+  color: "white",
+}
+let profPlum = {
+  firstName: "Victor",
+  lastName: "Plum",
+  occupation: "Designer",
+  age: 22,
+  description: "Billionaire video game designer",
+  image:
+    "https://vignette.wikia.nocookie.net/cluedo/images/a/a4/2016C6.jpg/revision/latest?cb=20200927103539",
+  color: "purple",
+}
+let missScarlet = {
+  firstName: "Kasandra",
+  lastName: "Scarlet",
+  occupation: "Actor",
+  age: 31,
+  description: "She is an A-list movie star with a dark past",
+  image: "https://www.radiotimes.com/uploads/images/Original/111967.jpg",
+  color: "red",
+}
+let mrsPeacock = {
+  firstName: "Eleanor",
+  lastName: "Peacock",
+  occupation: "Socialité",
+  age: 36,
+  description:
+    "She is from a wealthy family and uses her status and money to earn popularity",
+  image:
+    "https://vignette.wikia.nocookie.net/cluedo/images/b/b0/2016C5.jpg/revision/latest?cb=20200927103538",
+  color: "blue",
+}
+let mrMustard = {
+  firstName: "Jack",
+  lastName: "Mustard",
+  occupation: "Retired Football player",
+  age: 62,
+  description:
+    "He is a former football player who tries to get by on his former glory",
+  image:
+    "https://static.independent.co.uk/s3fs-public/thumbnails/image/2016/07/04/08/unspecified-3.jpg",
+  color: "yellow",
+}
+
+const suspectsArray = [
+  mrGreen,
+  drOrchid,
+  profPlum,
+  missScarlet,
+  mrsPeacock,
+  mrMustard
+]
 
-const suspectsArray = [];
 
 // Rooms Array
 
-const roomsArray = [];
+let diningRoom = {
```

Comment:
```suggestion
 {name: "Dining Room"},
 {name: "Conservatory"},
    {name: "Kitchen"},
```
inside of rooms array and you are not to set the variables for all the rooms the same with the others